sfr_eff.c

```diff
@@ -262,8 +262,9 @@ void cooling_and_starformation(void)
 
     double totsfrrate, localsfr=0;
     int i;
+    /* FIXME: this is inaccurate if some particles are made into garbage. may want to propagate garbage flag to the slots.*/
```

Comment:
I think this is fine, because we want the SFR from the garbage particles. Generally they will have been turned into stars...

Reply:
I am propagating IsGarbage to the secondary slots. So we have the option to skip them.